_pages/coc.md

```diff
@@ -3,3 +3,48 @@ layout: page
 title: Code of Conduct
 permalink: /coc/
 ---
+
+## tl;dr
+Follow the golden rule: treat others in the way that you would want to be treated.
+If you have a conflict with another member, or in any way feel harassed, please contact an admin to discuss the situation.
+
+## General Conduct
+All devanooga members are expected to follow these guidelines in our community:
+
+- Avoid language that is unnecessarily offensive to other users
+- Come prepared to share ideas _and defend them_. We want a place that is fun and thought provoking.
+- You may see ideas that are contrary to yours and that is ok.
+- While it is ok to express contrary ideas, it is **NOT** ok to attack, defame, or otherwise harass another member. It **IS** ok (and encouraged) to criticize ideas that you disagree with.
+- Bring a positive attitude to all interactions.
+- Assume that everyone else is being light-hearted and acting in good faith unless demonstrated otherwise.
+
+## Conflict Resolution
+If you or any other member is being treated in a way that violates this CoC, please contact an administrator immediately. You can contact us via slack, the [contact form](/contact/) on this site, or by [email](mailto:admin@devanooga.com).
+Administrators will attempt to resolve the conflict in a manner that is accessible to the rest of the devanooga community, such as in a public channel. If open dialogue puts the privacy of one or more of the parties at risk, parts or all of the conflict resolution may be carried out in private.
+
+## Conduct of Administrators
+devanooga is organized by Chattanooga-area members. We want to foster a community that is driven and shaped by the members. With that in mind:
+
+- Administrators never act alone. Every action taken by an administrator is discussed with the larger group of admins before the action is taken.
```

Comment:
I'm mixed on this -- administrators should have some autonomy and discretion (especially for lower-level punishments or asking nicely to cut stuff out), I also don't want any issue to be met with a delay in remediation if an administrator is online at the time of the incident but cannot convene with other administrators, and I don't want any decision to be perceived as "well we decided it as a group and it's final" out the gate, if an admin makes a choice that isn't agreed with, I'd like the user to be able to bring it up with the rest of the team and it be seriously reconsidered.

Not sure if my idea is _better_ though, or how I'd phrase that.